From 4d34b379411f781a494b4775260d550ff61e22e4 Mon Sep 17 00:00:00 2001
From: Tom Gundersen <teg@jklm.no>
Date: Fri, 16 Mar 2012 01:34:10 +0100
Subject: [PATCH] Makefile: move binary and avoid empty dirs

Install to /usr/bin by default, and don't install an empty /etc.

v2: Adjust README to reflect the change.
---
 Makefile |   10 ++++------
 README   |    2 +-
 2 files changed, 5 insertions(+), 7 deletions(-)

diff --git a/Makefile b/Makefile
index 2150ed3..ae432fa 100644
--- a/Makefile
+++ b/Makefile
@@ -6,12 +6,10 @@ CC := gcc
 all: bootchartd
 
 install: bootchartd
-	mkdir -p $(DESTDIR)/sbin
-	mkdir -p $(DESTDIR)/etc
-	mkdir -p $(DESTDIR)/usr/share/doc/bootchart/
-	install -m0755 bootchartd $(DESTDIR)/sbin/
-	install -m0644 bootchartd.conf.example $(DESTDIR)/usr/share/doc/bootchart/
-	install -m0644 README COPYING $(DESTDIR)/usr/share/doc/bootchart/
+	install -d $(DESTDIR)/usr/bin
+	install -d $(DESTDIR)/usr/share/doc/bootchart
+	install -m0755 bootchartd $(DESTDIR)/usr/bin/
+	install -m0644 README COPYING bootchartd.conf.example $(DESTDIR)/usr/share/doc/bootchart/
 
 OBJS := log.o svg.o bootchart.o
 
diff --git a/README b/README
index fa1d730..1906abb 100644
--- a/README
+++ b/README
@@ -6,7 +6,7 @@ Bootchart - a 'startup' graphing tool
 Bootchart is a tool, usually run at system startup, that collects and graphs
 the CPU and disk load of the system as it works. The output of bootchart is
 an SVG graph. Normally, bootchart is invoked as `bootchartd` by the kernel
-by passing "init=/sbin/bootchartd" to the kernel. Bootchart will then fork
+by passing "init=/usr/bin/bootchartd" to the kernel. Bootchart will then fork
 init off to resume normal system startup, while monitoring and logging
 startup information in the background.
 
-- 
1.7.9.5

